packages/mediacenter/kodi/scripts/systemd-addon-wrapper

```diff
@@ -33,3 +33,17 @@ if [ -f "/storage/.kodi/addons/$1/system.d/$1.service" ] ; then
     systemctl start "$1.service"
   fi
 fi
+
+if [ ! -d /storage/.cache/module-overlays ] ; then
+  mkdir -p /storage/.cache/module-overlays
+fi
```

Comment:
This block can probably be dropped - the directory should have been created by module-overlays-setup on boot.

OTOH the wrapper script also creates /storage/.config/system.d which should exist on all systems as well, so if in doubt I'd say better keep it.